### PR TITLE
[ace] Upgrade to 7.0.11

### PR DESCRIPTION
**Pull request by jwillemsen**
* ports/ace/portfile.cmake:
    * ports/ace/vcpkg.json:

Upgrade to ACE 7.0.11

- #### What does your PR fix?

Version upgrade

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  As with the previous release

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by JonLiu1993** [comment]
@jwillemsen, Thanks for your pr, could you please fix the conflict files?

**Comment by jwillemsen** [comment]
Fixed merge conflicts

**Comment by JonLiu1993** [comment]
> Please consider migrating this function to the new functions:
> 
> ```
> vcpkg_extract_source_archive_ex -> vcpkg_extract_source_archive
> ```

@jwillemsen , please use the new function to replace the deprecated function

File: ports/ace/portfile.cmake [hunk 1]
```diff
@@ -21,7 +21,7 @@ else()
     )
 endif()
 
-vcpkg_extract_source_archive_ex(
+vcpkg_extract_source_archive(
     OUT_SOURCE_PATH SOURCE_PATH
```
Comment (JonLiu1993) [inline ports/ace/portfile.cmake:25]:
```suggestion
    OUT_SOURCE_PATH SOURCE_PATH
```
```suggestion
    SOURCE_PATH
```

File: ports/ace/vcpkg.json [hunk 2]
```diff
@@ -5,6 +5,16 @@
...
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ],
```
Comment (JonLiu1993) [inline ports/ace/vcpkg.json:17]:
```suggestion
```

Comment (JonLiu1993) [reply ports/ace/vcpkg.json:17]: These dependencies do not require

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

**Pull request merged**